Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -45,8 +48,33 @@ ABenchmarkGymGameMode::ABenchmarkGymGameMode()
 
 	NPCSToSpawn = 0;
 	SecondsTillPlayerCheck = 15.0f * 60.0f;
+}
 
-	RNG.Initialize(123456); // Ensure we can do deterministic runs
+void ABenchmarkGymGameMode::GenerateTestScenarioLocations()
+{
+	const float ROAM_RADIUS = 7500.0f;
+	{
+		FRandomStream PlayerStream;
+		PlayerStream.Initialize(123456); // Ensure we can do deterministic runs
```

Comment:
What about making this seed a factor of the ExpectedPlayers? So they are still deterministic, but player 1 doesn't always run through the same points.